Makefile

```diff
@@ -228,6 +228,11 @@ test-e2e-java:
 test-e2e-source:
 	ginkgo $(GINKGO_FLAGS) -focus="odo source e2e tests" tests/e2escenarios/
 
+# Run image e2e tests
```

Comment:
```Run image e2e tests``` -> ```Run supported images e2e tests```

Reply:
@prietyc123 ^

Reply:
> `Run image e2e tests` -> `Run supported images e2e tests`

I would say if we add unsupported image e2e test in future then IMO `Run image e2e tests` is the perfect comment. WDYT?

Reply:
Right, i could not realize in that prospect. ```Run image e2e tests``` would be more genetic. So we can keep this

Reply:
My mistake, Unfortunately i interpreted your comment wrongly.

This is a target in the make file which can only execute the supported images. So it should be specific of what it does ```Run supported images e2e tests``` . As part of unsupported image target is concern, we can create a separate target anyway.